contrib/builder/rpm/s390x/clefos-base-s390x-7/Dockerfile

```diff
@@ -0,0 +1,19 @@
+#
+# THIS FILE IS AUTOGENERATED; SEE "contrib/builder/rpm/s390x/generate.sh"!
+#
+
+FROM sinenomine/clefos-base-s390x
```

Comment:
That is true; these guys are not part of the CentOS organization (which is with Red Hat now), so they are not allowed to name it that way. Technically, they follow CentOS 1:1. CentOS themselves does not cover s390x today.